animeal/src/Flows/Profile/ProfileViewModel.swift

```diff
@@ -58,6 +60,28 @@ final class ProfileViewModel: ProfileViewModelProtocol {
     }
 
     // MARK: - Interaction
+
+    func handleBackButton() {
+        let canSave = modelActions.first(where: {
+            ($0 as? ProfileModelSaveAction)?.appearance.isEnabled == true
+        }) != nil
+        guard canSave else {
+            coordinator.move(to: .cancel)
+            return
+        }
+
+        let viewAlert = ViewAlert(
+            title: "Do you really want to discard the changes?",
```

Comment:
Do we want to localise this or have this string in any central place?

Reply:
Yeah, appreciate noticing it. I missed it, we have localization, will add it and update review.